DELEGATEDTARGETS.md

```diff
@@ -0,0 +1,253 @@
+
+### `RepositoryEditor`
+
+Anytime a user has access to the keys for the standard TUF roles (`snapshot.json`, `timestamp.json`, `targets.json`), `RepositoryEditor` should be used.
+`RepositoryEditor` allows the user to edit a repository, and create a complete set of signed metadata. 
+`RepositoryEditor` also has the ability to load in role metadata and add them to the delegation chain, or use it to update an existing role. 
+In order to allow editing of all targets, `RepositoryEditor` uses a modal design. 
+Upon loading a repo, the standard targets role will be the one that is edited. 
+To change change the editor to a new targets role, first, the `TargetsEditor` must be cleared using `sign_targets_editor()`. 
+This signs the metadata that was being edited and inserts it to its proper place in the delegated targets structure. 
+Next, `change_delegated_targets()` is used to create a `TargetsEditor` for the new targets. 
+After `change_delegated_targets()` is called, the following methods will perform actions on the newly selected targets role: `targets_version()`, `targets_expires()`,  `add_target()`, `add_target_path()`, `add_target_paths()`,  `delegate_role()`, and `add_role()`. 
+There are three ways to work with the delegations structure in a repository, use `delegate_role()` to create a new role from scratch. 
+`delegate_role()` should be used whenever a user has access to the keys for the standard TUF roles and the keys for the new role. `RepositoryEditor` provides two methods of loading an existing metadata file, `update_role()` should be called if the targets role has already been added. 
+If a new role has been created it can be added to the repository by using `add_role()`.
+
+### `TargetsEditor`
+
+`TargetsEditor` is used to edit targets metadata if a user doesn’t have access to the standard TUF roles. 
+Metadata created from `TargetsEditor` will always be signed, but will not create a TUF repository. 
+The metadata from `TargetsEditor` must be loaded using `RepositoryEditor` to create a complete signed repository. 
+To create a new targets metadata simply use `TargetsEditor::new()`. 
+A `TargetsEditor` can also be created from a repository by using `from_repo()` or from an existing targets using `from_target`s`()`. 
+To encourage keeping role metadata up to date, `TargetsEditor` throws out the targets old version and expiration. 
+They must be set using `version()` and `expires()` before signing the `TargetsEditor`. 
+`TargetsEditor` provides a variety of methods that can be used to update a targets role. 
+After the role has been update, it should be signed using `sign()`. 
+Signing a `TargetsEditor` creates a `SignedDelegatedTargets` containing metadata for the new role as well as metadata for any roles that were added to the repository.
+
+## Tools
+
+All tuftool delegated targets commands follow the form
+
+* `tuftool delegation --signing-role <signing_role> <subcommand>`
+
+## delegation Subcommands
```

Comment:
Do you want this to be "`delegation` Subcommands" (referring to the `delegation` subcommand)? or "Delegation Subcommands"? (as a section header)